core/src/wsv.rs

```diff
@@ -157,6 +159,68 @@ impl WorldStateView {
         tokens
     }
 
+    /// Return a set of permission tokens granted to this account not as part of any role.
+    pub fn account_own_permission_tokens(
+        &self,
+        account: &Account,
+    ) -> impl ExactSizeIterator<Item = PermissionToken> {
+        self.world
+            .account_permission_tokens
+            .get(account.id())
+            .map_or_else(Default::default, |permissions_ref| {
+                permissions_ref.value().clone()
+            })
+            .into_iter()
+    }
+
+    /// Return `true` if [`Account`] contains a permission token not associated with any role.
+    #[inline]
+    pub fn account_contains_own_permission(
```

Comment:
```suggestion
    pub fn account_contains_explicit_permission(
```

Reply:
Or maybe "inherent" instead of "explicit" would also work here, to signify that the permission is tied to account only?